galley/pkg/crd/validation/webhook.go

```diff
@@ -39,6 +40,58 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
+var (
```

Comment:
Minor nit: Could you put these in a "monitoring.go" file, to keep things organized? We've done this in Mixer:
https://github.com/istio/istio/blob/master/mixer/pkg/runtime/config/monitoring.go